### PR TITLE
10.0 smangukiya 13342 attrib value multicompany

### PR DESCRIPTION
**Pull request by smangukiya**
related task 13342

New module name: ursa_prod_attrib_value_multicompany

### CODE REVIEW

File: ursa_prod_attrib_value_multicompany/models/product_attribute.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 - TODAY, Ursa Information Systems
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class ProductAttributeValue(models.Model):
+    _inherit = "product.attribute.value"
```
Comment (max3903) [inline ursa_prod_attrib_value_multicompany/models/product_attribute.py]: @smangukiya The company field should be on the `product.attribute.value.line`:
https://docs.google.com/document/d/1kqgUXW48JQ-IhmU-eACXn1IR8iRTF7aT0_Ys8zk43v8/edit

**Review by max3903 — CHANGES_REQUESTED** [review]

File: ursa_prod_attrib_value_multicompany/models/product_attribute.py (hunk 1 above)
Comment (smangukiya) [reply ursa_prod_attrib_value_multicompany/models/product_attribute.py]: @max3903 : please recheck them, I have updated changes. Thanks!

**Review by max3903 — CHANGES_REQUESTED** [review]
@smangukiya 

1. There could be different default value for each company, so the constraint needs to be updated to include the company field. Right now, I get an error message: "Please select one, first selected option will be valid!" when setting a default value for another company.
2. I was able to set a default value as a user of the other company, but when I try to configure the product, I get the following error message: "Access Error. Document type: res.company, Operation: read". I added the permission to the "Sales / Own Document Only" group, but I still get the error.

**Comment by smangukiya** [comment]
@max3903: 
Point 1: **FIXED**
Point 2: **FIXED**

Updated Modules:
ursa_prod_attrib_value_multicompany,
product_configurator_refactor


Thanks

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@smangukiya :+1:

**Pull request merged**